source/tutorials/db-coll.txt

```diff
@@ -0,0 +1,279 @@
+.. _scala-db-coll:
+
+=========================
+Databases and Collections
+=========================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: code example, access database, access collection
```

Comment:
Suggestion:

I think "access database" and "access collection" are similar to headings which I think are keywords that we were advised to avoid.
Other keywords that could be good to include might be:
codec registry
uuid
read preference
read concern
write concern
bson document 
schema validation